libs/light/neopixel.ts

```diff
@@ -1354,3 +1327,24 @@ namespace light {
         }
     }
 }
+
+namespace LightAnimation {
+    //% blockImage=1
+    //% block="rainbow" whenUsed
+    export const Rainbow = new light.RunningLightsAnimation(0xff, 0, 0, 50);
```

Comment:
The problem with this design is that animations are singleton so if you run the same animation on multiple LED strips they will interfere. It actually has to be a factory function (which is better for laziness).

Reply:
The laziness is similar with the `whenUsed` attribute. You could still use the same object and maybe have a number or two for animation on the strip.